### PR TITLE
Added the cocoapods-search plugin to CocoaPods

### PR DESCRIPTION
**Pull request by emkosz**
Added the gem 'cocoapods-search' to the Gemfile
Added the dependency to the cocoapods.gemspec
Deleted the source code for search and the source code for the search specs

Fixing this issue: https://github.com/CocoaPods/cocoapods-search/issues/3
And the original issue: https://github.com/CocoaPods/CocoaPods/issues/3397

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ group :development do
   cp_gem 'cocoapods-stats',      'cocoapods-stats'
   cp_gem 'cocoapods-trunk',      'cocoapods-trunk'
   cp_gem 'cocoapods-try',        'cocoapods-try'
+  cp_gem 'cocoapods-search',     'cocoapods-search'
```
Comment (segiddins) [inline Gemfile]: can we move this above stats so things stay in alphabetical order?

File: cocoapods.gemspec [hunk 2]
```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'cocoapods-stats',      '~> 0.5.3'
   s.add_runtime_dependency 'cocoapods-try',        '~> 0.4.5'
   s.add_runtime_dependency 'cocoapods-trunk',      '~> 0.6.1'
+  s.add_runtime_dependency 'cocoapods-search',     '~> 0.0.1'
```
Comment (segiddins) [inline cocoapods.gemspec]: same as above ^

**Comment by emkosz** [comment]
Fixed to be in alphabetic order.

**Comment by segiddins** [comment]
Awesome! If you rebase this, we can probably include it in 0.39

**Comment by emkosz** [comment]
Fantastic! Should be rebased now.

**Comment by segiddins** [comment]
:+1:

File: Gemfile.lock [hunk 3]
```diff
@@ -78,7 +85,7 @@ PATH
       claide (~> 0.9.1)
       cocoapods-core (= 0.39.0.beta.3)
       cocoapods-downloader (~> 0.9.3)
-      cocoapods-plugins (~> 0.4.2)
```
Comment (orta) [inline Gemfile.lock]: is this an accident? I still see the reference below?

`   s.add_runtime_dependency 'cocoapods-plugins',    '~> 0.4.2'`

Comment (segiddins) [inline Gemfile.lock]: good catch -- looks like a bad merge

Comment (emkosz) [inline Gemfile.lock]: I don't see cocoapods-plugins in my Gemfile.lock

**Comment by segiddins** [comment]
You'll also need to remove the relevant `require` statements from `command.rb`

**Comment by emkosz** [comment]
Woop! All checks have passed :)

**Comment by segiddins** [comment]
:+1:

**Comment by segiddins** [comment]
uh oh ... https://rubygems.org/gems/cocoapods-search

**Comment by emkosz** [comment]
nooo... :(

**Comment by orta** [comment]
It's now free, I think we can now push it - https://rubygems.org/gems/cocoapods-search

**Comment by segiddins** [comment]
:+1: once the gem is released!

**Comment by emkosz** [comment]
Gem released :+1:

File: cocoapods.gemspec [hunk 4]
```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'xcodeproj',            '~> 0.27.1'
   s.add_runtime_dependency 'cocoapods-downloader', '~> 0.9.3'
   s.add_runtime_dependency 'cocoapods-plugins',    '~> 0.4.2'
+  s.add_runtime_dependency 'cocoapods-search',     '~> 0.0.1'
```
Comment (segiddins) [inline cocoapods.gemspec]: needs to be bumped to `0.1.0` now?

File: Gemfile.lock [hunk 5]
```diff
@@ -48,6 +48,13 @@ GIT
       nap
 
 GIT
+  remote: https://github.com/CocoaPods/cocoapods-search.git
+  revision: 4af63d57cf6a4746a94739eb15f74ffa073e95a2
+  branch: master
+  specs:
+    cocoapods-search (0.0.1)
```
Comment (emkosz) [inline Gemfile.lock]: I guess 0.1.0 here too?

File: Gemfile.lock [hunk 6]
```diff
@@ -79,6 +86,7 @@ PATH
       cocoapods-core (= 0.39.0.beta.3)
       cocoapods-downloader (~> 0.9.3)
       cocoapods-plugins (~> 0.4.2)
+      cocoapods-search (~> 0.0.1)
```
Comment (emkosz) [inline Gemfile.lock]: And here

**Comment by segiddins** [comment]
:shipit: :fireworks:

**Pull request merged**